src/iosMain/kotlin/ch/dreipol/dreimultiplatform/FileManager.kt

```diff
@@ -14,12 +31,48 @@ actual val FileIdentifier.fileName: String?
 actual val FileIdentifier.filePath: String?
     get() = this.url.path
 
+actual fun FileIdentifier.createDirectoriesIfNotExists() {
+    fileManagerWithException { fileManager, errorPtr ->
+        fileManager.createDirectoryAtURL(url, withIntermediateDirectories = true, attributes = null, error = errorPtr)
+    }
+}
+
+actual fun FileIdentifier.exists(): Boolean {
+    return filePath != null && NSFileManager.defaultManager.fileExistsAtPath(filePath!!)
```

Comment:
Könntest du noch den filePath in eine lokale Variable schreiben, damit du das force unwrap nicht brauchst?